libtsuba/include/tsuba/ParquetReader.h

```diff
@@ -76,6 +76,11 @@ class KATANA_EXPORT ParquetReader {
   ///   \param uri an identifier for a parquet file
   katana::Result<int64_t> NumRows(const katana::Uri& uri);
 
+  /// Get the possible sub files for the logical parquet file, will return
+  /// a original parquet file if there are no sub files
+  ///   \param uri an identifier for a parquet file
+  katana::Result<std::vector<std::string>> GetSubFiles(const katana::Uri& uri);
```

Comment:
Probably simpler just to make this `GetFiles` an have it always return the "top level" parquet file. (maybe it already does that in which case the comment is confusing :D)